features/org.eclipse.n4js.dependencies.ui.sdk/feature.xml

```diff
@@ -56,6 +56,10 @@ Contributors:
          id="org.eclipse.wst.xml_ui.feature"
          version="0.0.0"/>
 
+   <includes
+         id="org.eclipse.epp.mpc"
+         version="0.0.0"/>
```

Comment:
shouldn't this be added to the `N4JS__IDE.launch` ?

Reply:
done